contents/authors.md

```diff
@@ -19,10 +19,10 @@ authors:
   - {
       handle: yakko-majuri,
       name: Yakko Majuri,
-      role: Developer Experience,
+      role: Software Engineer,
```

Comment:
While we're here, maybe better off loading images from S3 or from the repo itself to avoid an extra third-party link?